app.dstack.yaml

```diff
@@ -0,0 +1,50 @@
+type: service
+
+name: aana-chat-with-video
+
+image: nvidia/cuda:12.3.2-cudnn9-devel-ubuntu22.04
```

Comment:
There is a problem for the other images other than runpod ones. you can not ssh to them and for the app we need to setup the ngrok so the UI can access them.
You did not add that part and based on the image name you can not add them afterwards. You need to open some port and install some app (I read them in runpod docs) so the ssh and keys are added to the container

Reply:
SSH works perfectly fine for me with this image.
But we don't need to do any manual setup if we deploy with dstack. ngrok is also not needed anymore since we can access it through dstack gateway like this: https://aana-chat-with-video.aana.sky.dstack.ai.

Reply:
We need something more private. Is it some restriction we can set to limit the access just ourselves?

Reply:
It's the same as ngrok, public endpoint. But dstack has an option to enable authorization: https://dstack.ai/docs/reference/dstack.yml/service/#auth

Reply:
But don't we send requests to the app directly from web app?

Reply:
We use ngrok between ui backend and sdk